dotnet/src/SemanticKernel.Skills/Skills.OpenAPI/Authentication/BasicAuthenticationProvider.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.SemanticKernel.Skills.OpenAPI.Authentication;
+
+/// <summary>
+/// Retrieves authentication content (e.g. username/password, API key) via the provided delegate and
+/// applies it to HTTP requests using the "basic" authentication scheme.
+/// </summary>
+public class BasicAuthenticationProvider
```

Comment:
Could we rename these to use Authorization in place of Authentication? e.g., BasicAuthorizationProvider. Or is this following a paradigm you see in MSAL/.NET?

Reply:
Upon further discussion, going to leave this as-is.